Sources/PackageModel/Manifest/Manifest.swift

```diff
@@ -877,17 +865,27 @@ extension Manifest {
         return (knownPackage: known, unknownPackage: unknown)
     }
 
-    /// Returns the set of package dependencies that are potentially guarded by traits, per target. This does not
-    /// calculate enabled and disabled dependencies enforced by traits.
-    public func traitGuardedDependencies(lowercasedKeys: Bool = false) -> [String: [String: Set<String>]] {
-        self.targets.reduce(into: [String: [String: Set<String>]]()) { depMap, target in
-            let traitGuardedTargetDependencies = target.dependencies.filter {
-                !($0.condition?.traits?.isEmpty ?? true)
-            }
-            traitGuardedTargetDependencies.forEach {
-                guard let package = lowercasedKeys ? $0.package?.lowercased() : $0.package else { return }
-                depMap[package, default: [:]][target.name, default: []].formUnion($0.condition?.traits ?? [])
-            }
+
+    /// Computes the list of target dependencies per target that are guarded by traits.
+    /// A  target dependency is considered potentially trait-guarded if it defines a condition wherein there exists a list of traits.
```

Comment:
Formatting nit. Also, could the doc string contain a more detailed description for what the return type representst
```suggestion
    /// A target dependency is considered potentially trait-guarded if it defines a condition wherein there exists a list of traits.
```